charts/datasource-syncer/templates/datasource-syncer.yaml

```diff
@@ -0,0 +1,59 @@
+{{- /*
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+*/}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: datasource-syncer-init
+spec:
+  ttlSecondsAfterFinished: 60
+  template:
+    metadata:
+      labels:
+        app: datasource-syncer-init
+    spec:
+      containers:
+      - name: datasource-syncer-init
+        image: {{.Values.images.datasourceSyncer.image}}:{{.Values.images.datasourceSyncer.tag}}
+        args:
+        - "--datasource-uids=$DATASOURCE_UIDS"
+        - "--grafana-api-endpoint=$GRAFANA_API_ENDPOINT"
+        - "--grafana-api-token=$GRAFANA_API_TOKEN"
+        - "--project-id=$PROJECT_ID"
+      restartPolicy: Never
+---
+
+apiVersion: batch/v1
+kind: CronJob
```

Comment:
By convention, each resource should be in its own file and named according to the kind of resource it creates.

https://helm.sh/docs/chart_best_practices/templates/#structure-of-templates

Reply:
Happy to split